src/Semantic/OnlineIssueTooManyDNSLookups.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SPFLib\Semantic;
+
+use SPFLib\OnlineDnsLookup;
+use SPFLib\Record;
+
+class OnlineIssueTooManyDNSLookups extends OnlineIssue
```

Comment:
What about using
```php
SPFLib\Semantic\OnlineIssue\TooManyDNSLookups
```
instead of
```php
SPFLib\Semantic\OnlineIssueTooManyDNSLookups
```
?